api/src/main/java/org/hawkular/inventory/api/Inventory.java

```diff
@@ -26,4 +34,83 @@
 public interface Inventory extends AutoCloseable {
 
     Tenants.ReadWrite tenants();
+
+    /**
+     * Quickly navigate to the provided tenant.
+     *
+     * @param tenant the tenant to steer to.
+     * @return the access interface to the tenant
+     * @throws EntityNotFoundException if the tenant is not found
+     */
+    default Tenants.Single steerTo(Tenant tenant) throws EntityNotFoundException {
```

Comment:
maybe the above could even be folded into the `Inventory` interface after all:
```
inventory.inspect(env)
```
does that still look good to you as it does to me? :wink: 

Of course to make sense, the `inspect` methods should be default on the `Inventory` interface so that impls don't have to bother with it (if they don't want to).